### PR TITLE
Update

### PR DESCRIPTION
**Pull request by susiefen**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1943)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
@casey-collab, please try again

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
@casey-collab, try again?

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
@casey-collab

File: english/lighter-use/lighter-use.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+The marshmallow man explains that he thinks the lighter will 
+melt the walls of the prison so you can both escape. He says that 
+the lighter will slowly melt the marshmallow, but you must not put it
+too close to the wall. If the flame touches it, the wall will 
+burst into flames and you will both be consumed in the inferno.
+
+You:
+
+[Use the lighter to melt the wall](melt-wall/melt-wall.md)
```
Comment (casey-collab) [inline english/lighter-use/lighter-use.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/lighter-use/lighter-use.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+[Tell him he's nuts](nuts/nuts.md)
```
Comment (casey-collab) [inline english/lighter-use/lighter-use.md]: This link is also invalid.

Comment (casey-collab) [inline english/lighter-use/lighter-use.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/make-friends/make-friends.md [hunk 3]
```diff
@@ -15,3 +15,5 @@ Do you:
 [Have a cup of coffee?](../coffee/coffee.md)
 
 [Ask the marshmellow man, "What's it all for?"](question/question.md)
+
+[Ask what the lighter is for](lighter-use/lighter-use.md)
```
Comment (casey-collab) [inline english/make-friends/make-friends.md]: This link is also invalid.

File: english/melt-wall/melt-wall.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+You pick up the lighter and try to use it to melt the
```
Comment (casey-collab) [inline english/melt-wall/melt-wall.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/nuts/nuts.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+You tell him he is nuts. A lighter doesn't have
```
Comment (casey-collab) [inline english/nuts/nuts.md]: This page is also unreachable.

File: english/lighter-use/lighter-use.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+The marshmallow man explains that he thinks the lighter will
```
Comment (casey-collab) [inline english/lighter-use/lighter-use.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/melt-wall/melt-wall.md (hunk 4 above)
Comment (cbuckey-uda) [inline english/melt-wall/melt-wall.md]: If you put these new files inside the `lighter-use` directory that should fix your links, and I also think it's a nicer directory structure.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**